Sources/Apodini/Request/Response.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  File.swift
+//  
+//
+//  Created by Tim Gymnich on 24.11.20.
+//
+
+import Foundation
+import NIO
+@_implementationOnly import Vapor
+
+
+public protocol Response {
```

Comment:
Stupid question but why do we need a Response protocol?

Reply:
we use it to create a generic request handler (`Context.swift:24`) that for example takes a `Vapor.Request` and returns a `Vapor.Response`.

The protocol mainly defines how we encode responses.